### PR TITLE
✨ Add welcome page with submit button

### PR DESCRIPTION
**Pull request by fabiglahn**
## DESCRIPTION
The welcome page contains an image submit form.
The functionality will be implemented in further pull requests.

## DESIGN
Bildschirmfoto 2021-11-03 um 14 02 35

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -1,7 +1,31 @@
...
+    <main className={styles.container}>
+      <Title />
+      <div>
+        <h2 className={styles.explanation}>Welchen Song wünscht du dir?</h2>
+        <form className={styles.form}>
+          <input type="text" placeholder="mein name" />
+          <input type="text" placeholder="mein lieblingslied" />
+          <input type="text" placeholder="und das ist von..." />
+          <input
+            className={styles.button}
```
Comment (davidwobido) [inline src/app/App.tsx]:
```suggestion
            className={styles.submit}
```

File: src/app/App.module.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400&display=swap');
```
Comment (davidwobido) [inline src/app/App.module.css]: You could import the font in global.css and set the font-family for the whole project to 'Source Sans'

File: src/app/App.module.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.container {
+  display: grid;
+  grid-template-columns: 10% 45% 45%;
+  place-items: center;
+  font-family: 'Source Sans Pro', sans-serif;
```
Comment (davidwobido) [inline src/app/App.module.css]: see above @ font import

File: src/app/App.module.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  object-fit: cover;
+}
+
+.form {
+  display: grid;
+  row-gap: 0.5rem;
+}
+
+.button {
+  background-color: rgb(123 217 159);
```
Comment (davidwobido) [inline src/app/App.module.css]: You could set variables for the color in global.css

File: src/app/App.module.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  border: none;
+  padding: 0.3rem;
+  color: white;
+  text-align: center;
+  transition-duration: 0.5s;
+}
+
+.button:hover {
+  background-color: rgb(123 217 159 / 50%);
```
Comment (davidwobido) [inline src/app/App.module.css]: You could set variables for the color in global.css

File: src/app/App.module.css [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.explanation {
+  color: rgb(123 217 159);
```
Comment (davidwobido) [inline src/app/App.module.css]: You could set variables for the color in global.css

File: src/app/components/Title/title.module.css [hunk 7]
```diff
@@ -0,0 +1,9 @@
+.title {
+  text-transform: lowercase;
+  color: white;
+  text-align: center;
+  font-weight: 300;
+  font-family: 'Source Sans Pro', sans-serif;
```
Comment (davidwobido) [inline src/app/components/Title/title.module.css]: You could import the font in global.css and set the font-family for the whole project to 'Source Sans'

**Review by davidwobido — APPROVED** [review]
Looks good, just small suggestions

File: src/app/App.module.css [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+  height: 100vh;
+  object-fit: cover;
+}
+
+.form {
+  display: grid;
+  row-gap: 0.5rem;
+}
+
+.button {
```
Comment (welschoff) [inline src/app/App.module.css]: Maybe use "submit" as your className

**Review by manusmd — CHANGES_REQUESTED** [review]
lgtm 🔥

**Review by manusmd — APPROVED** [review]
lgtm🔥

**Pull request merged**